scripts/apps/authoring/authoring/controllers/AssociationController.ts

```diff
@@ -18,60 +18,12 @@ export function AssociationController(config, content, superdesk,
     mediaIdGenerator, authoring, renditions, notify) {
     const self = this;
 
-    /**
-     * @ngdoc method
-     * @name AssociationController#isMediaEditable
-     * @public
-     * @description Check if featured media can be edited or not. i.e. metadata/crops can be changed or not.
```

Comment:
Don't remove the comment if it contains details which can't be inferred from the function signature. For example, this one mentions crops, which isn't mentioned in the signature nor in the function itself. Can you keep the comments that you didn't yourself add?